### ISSUE TITLE
Benchmark bad variable name

### ISSUE THREAD

**Issue by hscstudio** [kenjis/php-framework-benchmark#51]
I just do this benchmark on my droplet Digital Ocean, I have git clone, and do
```
sh setup.sh laravel-5.1/ ci-3.0/ yii-2.0/
```
Success, but when I do
```
sh benchmark.sh laravel-5.1/ ci-3.0/ yii-2.0/
```
produce following error
```
benchmark.sh: 12: export: ci-3.0/: bad variable name
```

**Comment by kenjis** [comment]
I don't know why only `benchmark.sh` produce the error.
It seems they are using the same code.

How about this?
~~~
sh benchmark.sh laravel-5.1 ci-3.0 yii-2.0
~~~

**Comment by hscstudio** [comment]
It's same. produce the erorr too.

now, I comment and uncomment file list.sh, so I can benchmark specific framework only.

**Comment by cebe** [comment]
@hscstudio can you please check, if my pull request (#52) fixes it?

**Comment by hscstudio** [comment]
still produce same error

**Comment by kenjis** [comment]
What shell do you use?

**Comment by hscstudio** [comment]
I use putty to acces my droplet in DO

**Comment by kenjis** [comment]
@hscstudio No, no. It is not your shell. It is your terminal application or ssh client application.

Please run the command below in DO environment:
```
$ echo $SHELL
```
You can see the shell which you use.

**Comment by Fly06-Fr** [comment]
First at all, thank you for providing us with this benchmarking tool, this is really usefull.

I just wanted to mention that I run into the same problem and managed to solved it by using the right shell command (so "bash benchmark.sh ..." instead of "sh benchmark.sh ..." in my case).

http://www.linuxquestions.org/questions/linux-newbie-8/a-simple-bash-command-causing-bad-variable-name-790082/

**Comment by kenjis** [comment]
@Fly06-Fr 
Thank you for your information!

@hscstudio 
Please try this:
~~~bash
$ bash benchmark.sh laravel-5.1/ ci-3.0/ yii-2.0/
~~~

**Comment by Fly06-Fr** [comment]
@kenjis :

**Comment by Fly06-Fr** [comment]
Hi Kenjis,

Memory statistics are always null in the table summary.

Any clue about this?

Thanks.

**Comment by kenjis** [comment]
You can't get them, if you don't use Apache.

**Comment by Fly06-Fr** [comment]
I'm using Apache.

So what's wrong?

**Comment by kenjis** [comment]
Do you see memory statistics when you see hello world pages with your brower?

**Comment by Fly06-Fr** [comment]
My fault sorry.

I didn't have the correct DocumentRoot in my Apache conf. It's ok now.

I have a last question: How should I proceed to add another framework to your benchmarking tool?

**Comment by kenjis** [comment]
See source code of existing frameworks, please.
For example, https://github.com/kenjis/php-framework-benchmark/commits/master/ci-4.0-dev
